eve/workers/master/pod.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: worker
+spec:
+  containers:
+    - name: master
+      image: "{{ images.master }}"
+      resources:
+        requests:
+          cpu: "250m"
+          memory: 1Gi
+        limits:
+          cpu: "1"
+          memory: 1Gi
+      volumeMounts:
+        - name: worker-workspace
+          mountPath: /home/eve/workspace
+    - name: mongo
+      image: scality/ci-mongo:3.6.8
+      imagePullPolicy: IfNotPresent
```

Comment:
From what I'm understanding you're changing the CI and local environments to use an actual mongodb host instead of a "in memory" server/library. (correct me if wrong)

What I wonder is the following:
* Is this explicit when a dev wants to work on s3utils? Or should it be listed somewhere in the readme to start a mongo db server on the side. For me, I see this on the `main.yml` I get it, but I wonder what is the answer for others.
* As this is a "new" change and I see this `scality/ci-mongo` beeing used a lot, does it reflect the version of mongo that s3utils will speeks to? Or should we consider upgrading it a bit, to an upstream image per example (under the namespace `mongo`  on dockerhub) worth noting that this comment may be out of scope.

Reply:
@tcarmet 
> From what I'm understanding you're changing the CI and local environments to use an actual mongodb host instead of a "in memory" server/library. (correct me if wrong)

yes. The "in-memory" mongo introduces some flakiness, in which a teardown is not completed (port is not released), causing subsequent tests using mongo to fail.
 
> Is this explicit when a dev wants to work on s3utils? Or should it be listed somewhere in the readme to start a mongo db server on the side. For me, I see this on the main.yml I get it, but I wonder what is the answer for others.

Can you clarify what you mean by explicit? 
For development/testing, a mongodb server will be needed.

> As this is a "new" change and I see this scality/ci-mongo beeing used a lot, does it reflect the version of mongo that s3utils will speeks to? Or should we consider upgrading it a bit, to an upstream image per example (under the namespace mongo on dockerhub) worth noting that this comment may be out of scope.

Using the upstream mongodb image is great idea. `scality/ci-mongo:3.6.8` was used to be consistent with what is used in zenko 1.2.x and cloudserver project.